geospaas/base_viewer/forms.py

```diff
@@ -1,74 +1,43 @@
 from django.contrib.gis import forms
 from django.db.models import Q
+from django.forms import CharField
 from django.utils import timezone
 from leaflet.forms.widgets import LeafletWidget
 
-from geospaas.base_viewer.models import SearchModelForLeaflet
-from geospaas.catalog.models import Dataset as CatalogDataset
+from geospaas.catalog.models import Source as CatalogSource
 
-#from geospaas.base_viewer.models import Search_with_para
 
-class BaseForm(forms.ModelForm):
-    initial = dict()
+class OverallForm(forms.Form):
```

Comment:
Can you call it BaseForm? 
```suggestion
class BaseForm(forms.Form):
```